iceoryx2-ffi/ffi/src/api/unique_notifier_id.rs

```diff
@@ -86,6 +86,24 @@ impl HandleToType for iox2_unique_notifier_id_h_ref {
 
 // BEGIN C API
 
+#[no_mangle]
+unsafe extern "C" fn iox2_unique_notifier_id_value(
+    handle: iox2_unique_notifier_id_h,
+    id_ptr: *mut u8,
+) {
+    handle.assert_non_null();
```

Comment:
Could you also add a `debug_assert!(!id_ptr.is_null());`

This applies to all `unique_***_id_value()` functions!